src/main/java/com/notably/logic/parser/OpenCommandParser.java

```diff
@@ -34,12 +34,17 @@ public OpenCommandParser(Model notablyModel) {
         ArgumentMultimap argMultimap =
                 ArgumentTokenizer.tokenize(args, PREFIX_TITLE);
 
+        String titles;
         if (!ParserUtil.arePrefixesPresent(argMultimap, PREFIX_TITLE)
                 || !argMultimap.getPreamble().isEmpty()) {
-            throw new ParseException(String.format("Invalid input"));
+            titles = args.trim();
+            if (titles.equals("")) {
+                throw new ParseException("Path cannot be empty");
+            }
```

Comment:
You should actually do `title.isEmpty()`, more idiomatic!

Reply:
will do